Bottom: bcd24159552e2b1b18291f79befe2426ab0dd19f
Top:    4c785c52a48acf7c15898cbe0a8801c205188ed7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 23:15:19 +0000

Temp


---

diff --git a/src/polygon.c b/src/polygon.c
index 2fc09df..6a6d29a 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2385,6 +2385,8 @@ POLYAREA *board_outline_poly (bool include_holes)
   GList *pieces_to_delete = NULL;
   bool any_pieces_kept = false;
 
+  return NULL;
+
 #define BLOAT_WORLD MIL_TO_COORD (10)
 
   whole_world = RectPoly (-BLOAT_WORLD, BLOAT_WORLD + PCB->MaxWidth,
